.env

```diff
@@ -1,5 +1,2 @@
-export default{
-  jwt : {
-    secreKey : 'AppSecretKey1'
-  }
-}
\ No newline at end of file
+
+    secret = cycle10
```

Comment:
Blank line required and .env file should be ignored, you can’t push it to public

Reply:
yes I'm going to put it into gitignore